TESTS/mbed_hal/ospi/flash_configs/MX25LM51245G_config.h

```diff
@@ -0,0 +1,222 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018-2018 ARM Limited
```

Comment:
fix the year , 2020 and add please SPDX identifiers (all new files should have it, see platform/ files for instance)

Reply:
OK